tests/mvrf/temp_http_server.py

```diff
@@ -0,0 +1,28 @@
+import sys
+
+from BaseHTTPServer import HTTPServer
+from BaseHTTPServer import BaseHTTPRequestHandler
```

Comment:
Can we combine these two?
```suggestion
from BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
```

Reply:
Updated.